includes/definitions/ssu2000.yaml

```diff
@@ -0,0 +1,49 @@
+os: ssu2000
+text: 'Microsemi Synchronization Supply Unit'
+type: appliance
+icon: microsemi
+mib_dir: microsemi
+discovery:
+    -
+        sysObjectID:
+            - .1.3.6.1.4.1.601.3.1.
+discovery_modules:
```

Comment:
Hi @craig-nokia 
Any good reason to disable manually so many discovery modules ?

Reply:
Hi PipoCanaja,

The platform is just a bunch of specialized modules running under a communication processor.  I would guess for specialized hardware support, there is no reason to take resources for unnecessary discovery/polling.  I don't want to hammer the device  :)

Reply:
Usually, a discovery (when not supported by the device) only costs 1 or 2 snmp requests ... So you could very well leave it open, unless the SNMP support of the device is suffering from it (like an SNMP server that would crash if you request a non existing OID. ).
So you could probably remove that code without any impact on the device nor on the discovery/polling time.

Reply:
It's not a big deal to leave as I'm just trying to keep unnecessary traffic off the 15+ year old device.  I think I'm the only one  using this platform with librenms.    BTW, I did this same thing for the GE Pulsar power system several weeks ago.

Reply:
BTW, it was the polling that hammers.  Not the discovery.